src/standard_library/field_methods.md

```diff
@@ -0,0 +1,90 @@
+
+# Field
+
+After declaring a Field, you can use these common methods on it [^migrationNote]:
+
+## to_le_bits
+
+Transforms the field into an array of bits, Little Endian.
+
+``` rust
+fn to_le_bits<N>(_x : Field, _bit_size: u32) -> [u1; N]
+```
+
+example:
+
+``` rust
+fn main() {
+    const field = 2
+    let bits = field.to_le_bits(32);
+}
+```
+
+## to_le_bytes
+
+Transforms into an array of bytes, Little Endian
+
+``` rust
+fn to_le_bytes(_x : Field, byte_size: u32) -> [u8]
+```
+
+example:
+
+```rust
+fn main() {
+    const field = 2
+    let bytes = field.to_le_bytes(4);
+}
+```
+
+## to_le_radix
+
+Decomposes into a vector over the specificed base, Little Endian
+
+```rust
+fn to_le_radix(_x : Field, _radix: u32, _result_len: u32) -> [u8]
+```
+
+example:
+
+```rust
+fn main() {
+    const field = 2
```

Comment:
This is invalid syntax, we can use `let field = 2;` to declare a new variable

Reply:
Damn javascript, I'm so used to `const` now